reference/tools/pd-control.md

```diff
@@ -906,15 +879,20 @@ Encoding 格式示例：
 {{< copyable "" >}}
 
 ```bash
->> scheduler show                             // 显示所有的 schedulers
->> scheduler add grant-leader-scheduler 1     // 把 store 1 上的所有 Region 的 leader 调度到 store 1
->> scheduler add evict-leader-scheduler 1     // 把 store 1 上的所有 Region 的 leader 从 store 1 调度出去
->> scheduler add shuffle-leader-scheduler     // 随机交换不同 store 上的 leader
->> scheduler add shuffle-region-scheduler     // 随机调度不同 store 上的 Region
->> scheduler remove grant-leader-scheduler-1  // 把对应的 scheduler 删掉
+>> scheduler show                                 // 显示所有的 schedulers
+>> scheduler add grant-leader-scheduler 1         // 把 store 1 上的所有 Region 的 leader 调度到 store 1
+>> scheduler add evict-leader-scheduler 1         // 把 store 1 上的所有 Region 的 leader 从 store 1 调度出去
+>> scheduler add shuffle-leader-scheduler         // 随机交换不同 store 上的 leader
+>> scheduler add shuffle-region-scheduler         // 随机调度不同 store 上的 Region
+>> scheduler remove grant-leader-scheduler-1      // 把对应的 scheduler 删掉
+>> schedule pause balance-region-scheduler 10     // 暂停运行 balance-region 调度器 10 秒
+>> schedule pause all 10                          // 暂停运行所有的调度器 10 秒
+>> schedule resume balance-region-scheduler       // 继续运行 balance-region 调度器 
+>> schedule resume all                            // 继续运行所有的调度器 
+>> scheduler config balance-hot-region-scheduler  // 显示 balance-hot-region 调度器的配置
```

Comment:
It will be better to add parameter description about `src-tolerance-ratio` and `dst-tolerance-ratio` at the least